### DISCUSSION TITLE
Orphan visits - clear them permanently

### DISCUSSION THREAD

**Discussion by f8projects** [shlinkio/discussions#101] [Help wanted] [answered]
is it possible to delete all Orphan visits logs from database?

**Comment by acelaya** [comment, marked as answer]
Unfortunately no, but you can disable any further orphan visits tracking by providing `TRACK_ORPHAN_VISITS=false`, or via `php vendor/bin/shlink-installer set-option` (one or the other, depending how you are using Shlink).
